Crear preguntaalternativa

robreznor opened the pull request.

nachooooooox committed:
Merge remote-tracking branch 'refs/remotes/TeclerasUV-NodeJS/master' into Crear-Preguntaalternativa

nachooooooox committed:
Merge remote-tracking branch 'refs/remotes/TeclerasUV-NodeJS/master' into Crear-Preguntaalternativa

jorgegomezv committed:
Merge remote-tracking branch 'refs/remotes/TeclerasUV-NodeJS/master' into Crear-Preguntaalternativa

jorgegomezv committed:
Crear Editar Alternativa

Transnoche

jorgegomezv committed:
Modificar Footer

nachooooooox committed:
Merge remote-tracking branch 'refs/remotes/TeclerasUV-NodeJS/master' into Crear-Preguntaalternativa

jorgegomezv committed:
Subir Imagen

nachooooooox committed:
Merge branch 'Crear-Preguntaalternativa' of https://github.com/nacho7/TeclerasUV-NodeJS-Naxo into Crear-Preguntaalternativa

jorgegomezv committed:
Modificaciones Vistas

jorgegomezv committed:
Merge branch 'Crear-Preguntaalternativa' of https://github.com/nacho7/TeclerasUV-NodeJS-Naxo into Crear-Preguntaalternativa

Commit:
crear pregunta alternativa

nachooooooox committed:
modificar controlador editar

nachooooooox committed:
modificar pregunta alternativa

erickfmm commented:
¿?
Se supone que el tester manda el pull request.
Además, por favoooor, pongan descripción, yo no sé qué tiene esto, si está listo, testeado, o nada ._.

Commit:
Insertar respuestas

Se añadió  la inserción de las respuestas a la pregunta realizada por el
profesor. Falta ingresarle la ID de la pregunta maestra a la respuesta y
además agregarle si es correcta o no.

robreznor commented:
Sorry erick, se supone q el pull era para el nacho, aun tengo el problema q se envían a tu repo, hare otra vez el fork mejor.
